### PR TITLE
Make compatible with stable version of CommomMark

### PR DESCRIPTION
**Pull request by henriquemoody**
Related to #1

### CODE REVIEW

File: tests/spec/markua/aside.txt [hunk 1]
```diff
@@ -14,22 +12,18 @@ Run-on Aside
...
 
 === Name ===
 Multi-line Aside
 === Markdown ===
-A> An aside  
+A> An aside
 A> With multiple lines
 === Result ===
-<aside>
-<p>An aside<br />
```
Comment (henriquemoody) [inline tests/spec/markua/aside.txt:30]: I shouldn't remove this...

**Comment by mnapoli** [comment]
FYI I've given you access to the repo, you can merge yourself and also create branches in this repo

**Comment by henriquemoody** [comment]
Cool!

Thanks, @mnapoli!

**Comment by henriquemoody** [comment]
@mnapoli, can you enable Scrutinizer?

**Comment by mnapoli** [comment]
Done!

https://scrutinizer-ci.com/g/CouscousPHP/markua/ (first inspection in progress)

**Comment by henriquemoody** [comment]
Thanks!

**Pull request merged**